editoast/src/infra.rs

```diff
@@ -18,7 +19,7 @@ pub struct Infra {
     pub name: String,
     pub version: String,
     pub generated_version: Option<String>,
-    pub locked: bool,
+    pub locked: bool,    
```

Comment:
You might want to run a `cargo fmt` the CI is blocking you anyway.

Reply:
I do not get the point. Cargo fmt is ok for me

Reply:
Ok I get the point

Reply:
cargo clippy also